From 85e5949ed7659f36fc09788d82166f803aa95007 Mon Sep 17 00:00:00 2001
From: mkg20001 <mkg20001@gmail.com>
Date: Mon, 17 Jul 2017 21:32:03 +0200
Subject: [PATCH] Patch to work as module (hacky, hacky, hacky)

---
 bunyan | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/bunyan b/bunyan
index 330e25a..7f3839b 100755
--- a/bunyan
+++ b/bunyan
@@ -11,6 +11,8 @@
  * vim: expandtab:ts=4:sw=4
  */
 
+function Bunyan(process) {
+
 var VERSION = '1.8.10';
 
 var p = console.log;
@@ -1514,7 +1516,7 @@ function main(argv) {
     try {
         var opts = parseArgv(argv);
     } catch (e) {
-        warn('bunyan: error: %s', e.message);
+        console.warn('bunyan: error: %s', e.message);
         return drainStdoutAndExit(1);
     }
     if (opts.help) {
@@ -1650,3 +1652,9 @@ if (require.main === module) {
 
     main(process.argv);
 }
+
+main([])
+
+}
+
+module.exports = Bunyan
-- 
2.7.4

